src/Compilers/CSharp/Portable/Symbols/Source/SourceUserDefinedOperatorSymbolBase.cs

```diff
@@ -53,9 +52,30 @@ protected SourceUserDefinedOperatorSymbolBase(
 
             this.CheckUnsafeModifier(declarationModifiers, diagnostics);
 
-            if (isCompoundAssignmentOrIncrementAssignment)
```

Comment:
Is this covered by test?

Reply:
> Is this covered by test?

I am not sure what this question is about. There is no change at this position in the latest revision. Consider reviewing the latest revision. I had to do a merge to resolve a conflict. Apparently CodeFlow is now going crazy on merge commits and showing changes from them as undone in previous commits.